pallets/subtensor/src/errors.rs

```diff
@@ -50,67 +62,65 @@ mod errors {
         InvalidSeal,
         /// the dispatch attempts to set weights on chain with where any normalized weight is more than MaxWeightLimit.
         MaxWeightExceeded,
-        /// tempo is not valid.
-        InvalidTempo,
         /// the hotkey attempts to become delegate when they are already.
-        AlreadyDelegate,
+        HotKeyAlreadyDelegate,
         /// the hotkey attempts to set weights twice within net_tempo/2 blocks.
         SettingWeightsTooFast,
         /// a validator attempts to set weights from a validator with incorrect code base key.
         IncorrectNetworkVersionKey,
         /// an axon or prometheus serving exceeds the rate limit for a registered neuron.
         ServingRateLimitExceeded,
-        /// number of accounts going to be registered exceeds MaxAllowedUids for the network.
-        MaxAllowedUidsExceeded,
         /// the caller attempts to set weights with more uids than allowed.
-        TooManyUids,
-        /// a transactor exceeds the rate limit for transactions.
-        TxRateLimitExceeded,
+        UidsLengthExceedUidsInSubNet,
+        /// a transactor exceeds the rate limit for add network tx.
+        NetworkTxRateLimitExceeded,
+        /// a transactor exceeds the rate limit for delegate tx.
+        DelegateTxRateLimitExceeded,
+        /// a transactor exceeds the rate limit for set or swap hot key.
+        HotKeySetTxRateLimitExceeded,
         /// a transactor exceeds the rate limit for stakes.
         StakeRateLimitExceeded,
         /// a transactor exceeds the rate limit for unstakes.
         UnstakeRateLimitExceeded,
         /// registration is disabled
-        RegistrationDisabled,
+        SubNetRegistrationDisabled,
         /// registration attempt exceeds allowed in interval
```

Comment:
The number of registration attempts exceeded the allowed number in the interval.